app/views/users/show.html.erb

```diff
@@ -11,9 +11,21 @@
 
 <main class="col-md-8 side-frame">
   <div class="blog-main">
-    <section class="mentor divider">
-      <h1 class="subject no-frame">Request A Mentor</h1>
-      <%= link_to "Request a Mentor", new_mentor_path, class: "btn large-frame" %>
+    <section class="mentor">
+      <div class="col-md-6">
+        <h3 class="subject no-frame">Request A Mentor</h3>
+        <%= link_to "Request a Mentor", new_mentor_path, class: "btn", id: "mentor-button" %>
+      </div>
+      <div class="col-md-6">
+        <h3 class="subject no-frame">Will you participate in peer matches this month?</h3>
+        <h4 style="margin-top:0; margin-bottom:28px;"><%= @user.is_participating_this_month ? "YES" : "NO"%>&nbsp;
```

Comment:
Good catch, I meant to move that once I had figured it out, and then totally forgot!